cluster-autoscaler/core/scaledown/eligibility/eligibility.go

```diff
@@ -49,6 +48,8 @@ type utilizationThresholdGetter interface {
 	GetScaleDownUtilizationThreshold(context *context.AutoscalingContext, nodeGroup cloudprovider.NodeGroup) (float64, error)
 	// GetScaleDownGpuUtilizationThreshold returns ScaleDownGpuUtilizationThreshold value that should be used for a given NodeGroup.
 	GetScaleDownGpuUtilizationThreshold(context *context.AutoscalingContext, nodeGroup cloudprovider.NodeGroup) (float64, error)
```

Comment:
Why keep this method? It's redundant now, I think we should remove it.

Reply:
Removed `GetScaleDownGpuUtilizationThreshold()`.